spec/unit/burndown_chart_spec.rb

```diff
@@ -492,12 +492,21 @@
   end
 
   describe '.plot' do
+    use_given_filesystem
+
     it 'sends joined parsed options to python script' do
       allow(described_class).to receive(:process_options).and_return(%w{ --test 1 --no-blah })
       allow(described_class).to receive(:plot_helper).and_return('mescript')
       expect(described_class).to receive(:system).with('python mescript 42 --test 1 --no-blah')
       described_class.plot(42, foo: 1, bar: 2)
     end
+
+    it 'saves image file in given location' do
+      dir = given_directory
```

Comment:
why do you need this variable? :confused:

Reply:
It is actually the temporary directory created for writing the test data ( .png file ) ,as  we are using GivenFileSystemSpecHelpers in tests. 
http://www.rubydoc.info/gems/given_filesystem/0.1.2/file/README.md#Thanks

Reply:
but why do you need to assign it to a variable? :confused: